lib/index.js

```diff
@@ -1,19 +1,41 @@
 var foodsRequests = require('./requests/foods')
+var foodsDiary = require('./requests/diaryFoods')
 require('./foods')
 
 $(document).ready(() => {
-  foodsRequests.getFoods()
+  var fileName = location.pathname.split('/').slice(-1)[0]
+
+  if (fileName === 'foods.html') {
```

Comment:
Interesting approach to specifying functions according to what page a user is navigating through!